cmd/crank/xpkg/push.go

```diff
@@ -28,20 +28,23 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/errors"
 	"github.com/crossplane/crossplane-runtime/pkg/logging"
 
-	"github.com/crossplane/crossplane/cmd/crank/internal/upbound/credhelper"
 	"github.com/crossplane/crossplane/internal/xpkg"
+	"github.com/crossplane/crossplane/internal/xpkg/upbound/credhelper"
 )
 
 const (
 	errGetwd           = "failed to get working directory while searching for package"
 	errFindPackageinWd = "failed to find a package in current working directory"
 )
 
+// DefaultRegistry for pushing Crossplane packages.
+const DefaultRegistry = "xpkg.upbound.io"
+
 // pushCmd pushes a package.
 type pushCmd struct {
 	fs afero.Fs
 
-	Tag     string `arg:"" help:"Tag of the package to be pushed. Must be a valid OCI image tag."`
+	Tag     string `arg:"" help:"Tag of the package to be pushed. Must be a valid OCI image tag. Unqualified tags will be pushed to xpkg.upbound.io."`
 	Package string `short:"f" help:"Path to package. If not specified and only one package exists in current directory it will be used."`
```

Comment:
```suggestion
	Tag     string `arg:"" help:"Package filename and version tag to use in the container registry."`
	Package string `short:"f" help:"Path to the Crossplane xpkg file." default:"."`
```

What do you mean by "Unqualified tags"?